src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java

```diff
@@ -87,6 +90,29 @@ public void testWithoutPublishers() throws Exception {
         reconfigure(p);
     }
     
+    @Bug(19985)
+    public void testNullCondition() throws  Exception {
+      FreeStyleProject p = createFreeStyleProject();
+      ConditionalPublisher conditionalPublisher = new ConditionalPublisher(new AlwaysRun(),
+              Collections.<BuildStep>singletonList(null),
+              null,
+              false,
+              null,
+              null);
+      FlexiblePublisher fp = new FlexiblePublisher(Arrays.asList(conditionalPublisher));
+      p.getPublishersList().add(fp);
+      p.save();
+      jenkins.reload();
```

Comment:
BTW another way of doing this is to use `@LocalData` with a prepared example of historically broken configuration.